Add 3 examples

bjorncs opened the pull request:
The two more complex examples are copied from vespa-http-client documentation.

I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

bjorncs committed:
Add 3 examples

The two more complex examples are copied from vespa-http-client documentation.

jonmv commented on vespa-feed-client/src/test/java/ai/vespa/feed/client/examples/JsonFileFeederExample.java:
```diff
@@ -0,0 +1,89 @@
+        public void onNextResult(Result result, Throwable error) {
+            resultsReceived.incrementAndGet();
+            if (error != null) {
+                log.warning("Problems with feeding document");
```
Let's remember to add doc id here.

jonmv commented on vespa-feed-client/src/test/java/ai/vespa/feed/client/examples/JsonStreamFeederExample.java:
```diff
@@ -0,0 +1,146 @@
+                    + " sent, success rate " + String.format(Locale.US, "%.2f docs/sec", rate) + ").");
+        }
+    }
+}
```
Not sure I like this example very much. Could we rewrite it to just use the `FeedClient` directly? That one blocks when you attempt to send new operations and there's no free inflight slots, which avoids blowing up memory.

bjorncs replied:
Agreed - the example is now simplified to just print result of `FeedClient.stats()` instead of manual book-keeping.

bjorncs committed:
Simplify JsonStreamFeederExample

Use FeedClient.stats() instead of manual book-keeping.
Include document id in error messages.

bjorncs commented:
@jonmv PTAL
